doc/release-notes-17437.md

```diff
@@ -1,5 +1,6 @@
 Low-level RPC Changes
 ===
 
-- The RPC gettransaction, listtransactions and listsinceblock responses now also
-includes the height of the block that contains the wallet transaction, if any.
+- The RPC gettransaction, listtransactions, and listsinceblock responses now
+include the height of the block that contains the wallet transaction, if any.
+Please refer to the RPC help of these calls for details.
```

Comment:
As usual? :smile:

Reply:
As per @laanwj suggestion in IRC a few months ago: `Release notes should refer to the RPC help for details instead of substituting for full documentation. Example: "Please refer to the RPC help of getbalances for details."`

The commit also fixes a grammar error: s/includes/include/

Reply:
(Thanks for reviewing :)

Reply:
Maybe instead of mentioning it in every bullet point, once per heading is enough?

Reply:
Done